profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -427,6 +430,16 @@ function nexteuropa_multilingual_path_update($path) {
  *     - language: The language of the alias.
  */
 function _nexteuropa_multilingual_path_und_sync($path) {
+
+  // Check if entity has a translation mechanism other than entity translation.
+  $node = menu_get_object("node", 1, $path['source']);
```

Comment:
Concerning the $path['source'], on working to another ticket, I observed that it was not always set. I do not remember precisely in which conditions. I hesitate between content types that do not use "content translation" nor "Entity translation" or those that do not use revision.